crocket/src/main/java/com/crocket/model/Player.java

```diff
@@ -48,6 +49,8 @@ public void handleEvent(PassTargetEvent event) {
         Entity target = event.getTarget();
         if (this.remainingTargets.peek() == target) {
             this.remainingTargets.poll();
+            this.incrementScore();
```

Comment:
What is score used for? Are we not just supposed to pass under them?

Reply:
Will fix